cartoframes/layer.py

```diff
@@ -184,15 +184,11 @@ class QueryLayer(AbstractLayer):
             values described:
 
             - column (str): Column to base coloring from.
-            - scheme (str, optinal): Color scheme from
-              `CartoColors
+            - scheme (dict, optional): Dictionary definition of color scheme,
+              or a scheme such as `styling.sunset(7)` from the
+              styling module of cartoframes that exposes `CartoColors
               <https://github.com/CartoDB/CartoColor/wiki/CARTOColor-Scheme-Names>`__.
-              Defaults to `Mint`.
-            - bin_method (str, optional): Quantification method for dividing
```

Comment:
These other keys are still possible. A user could do
```python
cc.map(layers=Layer(..., color={'column': 'axolotl',
                                'scheme': dict(bin_method='quantiles', bins=7, name='Vivid')}))
```

And actually, there might be a hidden 'colors' key somewhere you can fill custom values!